cfy_manager/config.py

```diff
@@ -81,9 +94,27 @@ def dump_config(self):
                     )
                 )
 
+    def _apply_forced_settings(self):
+        # We should get rid of any settings we're forcing here when we can,
```

Comment:
do you mean this as a todo? if so mark it 'TODO' at least

Reply:
It's not really a TODO that applies to here I think? It's more of a note that if you're adding something to this section you should think carefully about whether the actual config var it applies to makes sense to have.

Reply:
Comment removed anyway- agreed that it was unclear whether it was a call to action or a call to contemplation, so it's not very useful in that form.